### PR TITLE
RHDEVDOCS 5933 fix spec in configuring PAC

### PR DESCRIPTION
**Pull request by mramendi**
Version(s):

please cp to pipelines-docs-1.11, pipelines-docs-1.12, pipelines-docs-1.13, pipelines-docs-1.14

Issue:

RHDEVDOCS 5933

Link to docs preview:

https://74504--ocpdocs-pr.netlify.app/openshift-pipelines/latest/pac/install-config-pipelines-as-code.html#customizing-pipelines-as-code-configuration_install-config-pipelines-as-code

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by mramendi** [comment]
https://issues.redhat.com/browse/RHDEVDOCS-5933

File: modules/op-customizing-pipelines-as-code-configuration.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@
 = Customizing {pac} configuration
 
 [role="_abstract"]
-To customize {pac}, cluster administrators can configure the following parameters in the `TektonConfig` custom resource, in the `pipelinesAsCode.settings` spec:
+To customize {pac}, cluster administrators can configure the following parameters in the `TektonConfig` custom resource, in the `platforms.openshift.pipelinesAsCode.settings` spec:
```
Comment (piyush-garg) [inline modules/op-customizing-pipelines-as-code-configuration.adoc:9]: should we mention `spec.` also in start?

Comment (mramendi) [reply modules/op-customizing-pipelines-as-code-configuration.adoc:9]: Not really as the word "spec" implies it.

Comment (mramendi) [reply modules/op-customizing-pipelines-as-code-configuration.adoc:9]: Example of such use: https://docs.openshift.com/pipelines/1.14/create/remote-pipelines-tasks-resolvers.html#resolver-hub-config_remote-pipelines-tasks-resolvers

Other places do have "spec." but they call it "parameter".

It's a good question of consistency to raise on a higher level, I guess. I'll try to raise it. Meanwhile can we go on with this merge?

Comment (mramendi) [reply modules/op-customizing-pipelines-as-code-configuration.adoc:9]: I have raised it with other writers and the answers I got were generally in support of the way I did it

**Comment by savitaashture** [comment]
LGTM

**Review by ppitonak — APPROVED** [review]

**Review by xenolinux — COMMENTED** [review]
LGTM

/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label peer-review-done

**Pull request merged**